packages/libs/designer-decisions/src/types/primitive-values/color/channel.ts

```diff
@@ -0,0 +1,28 @@
+import type { ColorValue } from './color';
+import type {
+    ColorOklabChannelName,
+    ColorOklabChannelValue,
+    ColorOklabComplementaryChannels,
+} from './oklab';
+import type {
+    ColorSRGBChannelName,
+    ColorSRGBChannelValue,
+    ColorSRGBComplementaryChannels,
+} from './srgb';
+
+export type ColorChannelName = ColorSRGBChannelName | ColorOklabChannelName;
+
+export type ColorChannelValue = ColorSRGBChannelValue | ColorOklabChannelValue;
+
+export type ColorChannelValueCommon = Omit<
+    ColorSRGBChannelValue | ColorOklabChannelValue,
+    'toColor'
+> & {
+    toColor: (
+        channels: ColorSRGBComplementaryChannels | ColorOklabComplementaryChannels,
```

Comment:
Let's change the implementations as well

e.g.: `src/primitives/color/oklab-chroma-value/createOklabChromaValue.ts`